configloader/png/tests/test_config.py

```diff
@@ -0,0 +1,12 @@
+from dffml.util.asynctestcase import AsyncTestCase
+
+from dffml_config_png.configloader import PNGConfigLoader
+
+
+class TestConfig(AsyncTestCase):
+    async def test_dumpb_loadb(self):
+        async with PNGConfigLoader.withconfig({}) as configloader:
+            async with configloader() as ctx:
+                original = {"Test": ["dict"]}
+                reloaded = await ctx.loadb(await ctx.dumpb(original))
```

Comment:
Change this to just do `loadb`.

Reply:
I think you still need to make this only test the loadb function. I'd suggest using one of the images you added to the `examples/MNIST/` directory. You can find it with `pathlib.Path(__file__).parent / ".." / ".." / ".." / "examples" / "MNIST" / "image1.mnistpng"`.